index.html

```diff
@@ -2931,7 +2961,7 @@ <h3>Zero-Knowledge Proofs</h3>
           <li>
 Combine multiple <a>verifiable credentials</a> from multiple <a>issuers</a> into
 a single <a>verifiable presentation</a> without revealing
-<a>verifiable credential</a> or <a>subject</a> identifiers to the
+<a>verifiable credential</a> or <a>holder</a> identifiers to the
```

Comment:
You cannot replace `subject` with `holder` like this. It *might* work out if you added `holder` to `subject`, but only if you use `associated` or similar instead of `bound`; there may be other issues with new draft text.